atom-extension/grammars/mydsl.cson

```diff
@@ -0,0 +1,13 @@
+'scopeName': 'source.mydsl'
+'name': 'MyDslFiles'
+'fileTypes': [
+  'mydsl'
+]
+injections:
+  'L:*':
+    patterns: [
+        {
+            match: '\\b(Hello|from)\\b'
```

Comment:
Is this handwritten or generated?

Reply:
we dont have any fragments for such things yet.